Sources/Fuzzilli/Mutators/OperationMutator.swift

```diff
@@ -169,8 +169,18 @@ public class OperationMutator: BaseInstructionMutator {
             } else {
                 newOp = BeginFor(comparator: op.comparator, op: chooseUniform(from: allBinaryOperators))
             }
-        case let op as BeginAnyFunctionDefinition:
-            newOp = BeginAnyFunctionDefinition(signature: op.signature, isStrict: !op.isStrict)
+        case let op as BeginPlainFunctionDefinition:
+                newOp = BeginPlainFunctionDefinition(signature: op.signature, isStrict: !op.isStrict)
```

Comment:
Nit: wrong indention here and the line below

Reply:
fixed